content/docs/reference/neon-cli.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Neon CLI
+subtitle: Use the Neon CLI to manage Neon projects from the command line
+enableTableOfContents: true
+isDraft: true
+---
+
+Neon offers several methods for working with your projects. Utilizing the Neon Command Line Interface (CLI), you can operate Neon directly from a terminal or via automation. The Neon CLI facilitates numerous functions, such as Neon authentication, project creation and management, role assignment, and more.
+
+## Synopsis
+
+The `neonctl` command can be called from the command line. Without any arguments, it displays command usage and help:
+
+```bash
+neonctl --help
+usage: neonctl <cmd> [args]
+
+Commands:
+  neonctl auth               Authenticate
+  neonctl projects           Manage projects
+  neonctl me                 Show current user
+  neonctl branches           Manage branches
+  neonctl endpoints          Manage endpoints
+  neonctl databases          Manage databases
+  neonctl roles              Manage roles
+  neonctl operations         Manage operations
+  neonctl connection-string  Get connection string                 [aliases: cs]
+
+Options:
+      --version     Show version number                                [boolean]
+      --help        Show help                                          [boolean]
+  -o, --output      Set output format
+                  [string] [choices: "json", "yaml", "table"] [default: "table"]
+      --api-host    The API host   [default: "https://console.neon.tech/api/v2"]
+      --config-dir  Path to config directory
+                             [string] [default: "/home/dtprice/.config/neonctl"]
+      --oauth-host  URL to Neon OAUTH host [default: "https://oauth2.neon.tech"]
+      --client-id   OAuth client id                [string] [default: "neonctl"]
+      --api-key     API key                               [string] [default: ""]
+```
+
+## Install the Neon CLI
+
+This topic describes how to install the `neonctl` command-line interface tool and connect to Neon.
+
+### Prerequisites
+
+- Node.js 14.0 or higher. To check if you already have Node.js, run the following command:
```

Comment:
not sure about version 14, didn't try it. Let's put 16? Current LTS is 18

Reply:
updated as suggested.